test-harness/src/test/groovy/com/netflix/conductor/test/base/AbstractResiliencySpecification.groovy

```diff
@@ -30,8 +30,8 @@ import spock.mock.DetachedMockFactory
 @TestPropertySource(properties = [
         "conductor.system-task-workers.enabled=false",
         "conductor.workflow-repair-service.enabled=true",
-        "conductor.workflow-sweeper.enabled=true",
-        "conductor.integ-test.queueSpyEnabled=true"
+        "conductor.workflow-sweeper.enabled=false",
```

Comment:
Disabling the sweeper because repair service is invoked explicitly in the resiliency spec.